src/main/java/org/obsidiantoaster/generator/rest/ObsidianResource.java

```diff
@@ -264,10 +311,12 @@ public Response downloadZip(JsonObject content,
             Result result = controller.execute();
             if (result instanceof Failed)
             {
-               return Response.status(Status.INTERNAL_SERVER_ERROR).entity(result.getMessage()).build();
+               return Response.status(Status.INTERNAL_SERVER_ERROR).entity(getMessage(result)).build();
```

Comment:
what would `org.jboss.forge.service.util.UICommandHelper.describeResult(JsonObjectBuilder, Result)` do if the result.message was a JSON string already?  Only I kinda like the idea of the command deciding what the JSON output is; but it'd be cool if `describeResult` could combine multiple json results together. 

I guess longer term it'd be nice to be able to associate a Result entity really; then for a CompositeResult we'd return a List of the entities and then turn the whole thing into a JSON string using, say, jackson